ceph/utils.py

```diff
@@ -165,6 +165,33 @@ def volume_cleanup(volume, osp_cred):
                     return 1
 
 
+# Added this function to cleanup stale volumes with status deleting, available, error, unknown, reserved
+def cleanup_ceph_vols(osp_cred, pattern=None, timeout=300):
+    driver = get_openstack_driver(osp_cred)
+    timeout = datetime.timedelta(seconds=timeout)
+    for volume in driver.list_volumes():
```

Comment:
IMO, this won't be required, since we have another loop where it iterates volumes again. there we can have log with vol status.